myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -374,6 +380,166 @@ def ncol(self):
         return self.hfield.ncol
 
 
+class RepellerChallengeOpponent(ChallengeOpponent):
+    # Repeller parameters
+    DIST_INFLUENCE = 3.5 # Distance of influence by the repeller
+    ETA = 20.0 # Scaling factor
+    MIN_SPAWN_DIST = 1.5
+
+    def __init__(self, sim, rng, probabilities: list, min_spawn_distance: float, linear_velocity: float):
+        self.dt = 0.01
+        self.sim = sim
+        self.rng = rng
+        self.opponent_probabilities = [0.1, 0.35, 0.35, 0.2]
+        self.min_spawn_distance = min_spawn_distance
+        self.noise_process = pink.ColoredNoiseProcess(beta=2, size=(2, 2000), scale=10, rng=rng)
+        self.randomize_vel = True if linear_velocity == 'random' else False
+        self.linear_velocity = linear_velocity if type(linear_velocity) == float or type(linear_velocity) == int else 1.0
+        self.reset_opponent()
+
+    def get_agent_pos(self):
+        """
+        Get agent Pose
+        :param pose: Pose of the agent, measured from the pelvis.
+        :type pose: array -> [x, y]
+        """
+        return self.sim.data.body('pelvis').xpos[:2]
+
+    def get_wall_pos(self):
+        """
+        Get location of quad boundaries.
+        :param pose: Pose of points along quad boundaries.
+        :type pose: array -> [x, y]
+        """
+        bound_resolution = np.linspace(-8.7, 8.7, 25)
```

Comment:
hard coded numbers. 
- Best will be to get them automatically 
- At the very least, let's use a global variable for them